transform-plugins/src/main/java/io/cdap/plugin/XMLMultiParser.java

```diff
@@ -82,6 +85,14 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
     pipelineConfigurer.getStageConfigurer().setOutputSchema(config.getSchema(collector));
   }
 
+  @Override
+  public void prepareRun(StageSubmitterContext context) throws Exception {
+    super.prepareRun(context);
+    context.record(TransformLineageRecorderUtils
+      .oneToOneIn(new ArrayList<>(fieldNames), "multiParse",
```

Comment:
This is basically parse one xml field to all the output fields, config.field is the only field we use in the transform